CMakeLists.txt

```diff
@@ -18,15 +18,28 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 find_package(KOKKOS REQUIRED)
 
 option(Cabana_ENABLE_Serial "Build Cabana with Serial support" ON)
-find_package(OpenMP QUIET)
-option(Cabana_ENABLE_OpenMP "Build Cabana with OpenMP support" ${OpenMP_FOUND})
+
+option(Cabana_ENABLE_Pthread "Build Cabana with Pthread support" OFF)
+if( Cabana_ENABLE_Pthread )
+  find_package(Threads)
```

Comment:
Do we really need to find the thread package? Isn’t it inherited from Kokkos.

Reply:
I guess I would rather be thorough here

Reply:
Ok